EnglishTranslation-master/QUEST_LV_0200.tsv

```diff
@@ -3957,9 +3957,9 @@ QUEST_LV_0200_20150323_003956	So you are safe.{nl}I am going to go back to my ho
 QUEST_LV_0200_20150323_003957	Are the peaceful days going to return like the old times?{nl}Revelator..
 QUEST_LV_0200_20150323_003958	Revelator. I want to bury Dallas under the Ramybes Cliff where he can see his hometown.{nl}But there are so many monsters there. Please get rid of them.
 QUEST_LV_0200_20150323_003959	First, it would be better to find some documents at the Sanctuary.{nl}I remember there are documents there about methods of disguising as demons.
-QUEST_LV_0200_20150323_003960	Okay. Use the scroll to lure the demons to the Gaule Altar.{nl}If you want to know what will happen.. Well then you will have to wait and see.
+QUEST_LV_0200_20150323_003960	Okay. Use the scroll to lure the demons to the Apgaules Altar.{nl}If you want to know what will happen.. Well then you will have to wait and see.
```

Comment:
`Well then you will have to wait and see.`
to
`Well, you'll have to wait and see.`